Dockerfile_buildx_deb

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:24.04 AS base
+RUN apt-get update \
+ && apt-get install --no-install-recommends -y \
+     fakeroot=1.33-1 git=1:2.43.0-1ubuntu7.2 build-essential=12.10ubuntu1 \
+     devscripts=2.23.7 debhelper=13.14.1ubuntu5 dh-make=2.202304 \
+     btrfs-progs=6.6.3-1.1build2 markdown=1.0.1-12 pkg-config=1.8.1-2build1 \
+ && rm -rf /var/lib/apt/lists/*
+
+FROM base AS build
+WORKDIR /build/bees
+RUN chown -R ubuntu:ubuntu /build 
+USER ubuntu
+RUN git clone https://github.com/Zygo/bees.git .
+ARG DEBFULLNAME="Maintainer Name"
+ARG DEBEMAIL="usere@example.pro"
+RUN mkdir debian \
+ && printf "Source: bees\n\
+Section: utils\n\
+Priority: optional\n\
+Maintainer: %s <%s>\n\
+Build-Depends: debhelper-compat (= 13), markdown, btrfs-progs\n\
```

Comment:
if you're on a distro that has it (like Ubuntu or Debian), you really want `cmark-gfm`.  Last time I checked, OG markdown didn't support tables at all.